forum/projects/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+from rest_framework.mixins import ListModelMixin
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.response import Response
+from drf_yasg.utils import swagger_auto_schema
 
-# Create your views here.
+from .models import Industry
+from .serializers import IndustrySerializer
+
+
+class IndustryViewSet(GenericViewSet, ListModelMixin):
+    queryset = Industry.objects.all()
+
+    @swagger_auto_schema(
+            responses={
+                '200': IndustrySerializer
+            }
+    )
```

Comment:
Ensure the schema documentation includes correct details, such as the operation description and summary, eg.
```
    @swagger_auto_schema(
        operation_description="Retrieve a list of industries",
        operation_summary="List Industries",
        responses={
            '200': IndustrySerializer(many=True)
        }
    )
```